applications/omnipresence/src/omnipresence_maintenance.erl

```diff
@@ -11,11 +11,18 @@
 -include("omnipresence.hrl").
 
 -export([current_subscriptions/0, current_subscriptions/1, current_subscriptions/2
+         ,count_current_subscriptions/0
          ,subscribe/2
          ,send_mwi_update/3
          ,list_terminated_callids/0
         ]).
 
+-spec count_current_subscriptions() -> 'no_return'.
+count_current_subscriptions() ->
+    CurrentSubscriptions = ets:tab2list(omnip_subscriptions:table_id()),
+    io:format("~p\n", [length(CurrentSubscriptions)]),
```

Comment:
@fenollp @macpie please use `~n` for endlines, not `\n`. Let's not merge things too quickly!